api/proto/ci/travis.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package proto.ci;
+
+option go_package = "github.com/spinnaker/kleat/api/client/ci";
+
+import "permissions.proto";
+
+// Configuration for Travis.
+message Travis {
+  // Whether Travis is enabled.
+  bool enabled = 1;
+
+  // The list of configured Travis accounts.
+  repeated TravisAccount masters = 2;
+}
+
+// Configuration for a Travis account.
+message TravisAccount {
+  // The name of the account.
+  string name = 1;
+
+  // (Required) The address of the Travis API (https://api.travis-ci.org).
+  string address = 2;
+
+  // (Required) The base URL to the Travis UI (https://travis-ci.org).
+  string baseUrl = 3;
+
+  // The GitHub token with which to authenticate against Travis.
+  string githubToken = 4;
+
+  // (Required) The number of repositories the Travis integration should fetch
+  // from the API each time the poller runs. Should be set a bit higher than the
+  // expected maximum number of repositories built within the poll interval.
+  int32 numberOfRepositories = 5;
```

Comment:
Another one where 0 vs. unset matters, which I've dealt with (for now) by marking it as required.